include/fuse_lowlevel.h

```diff
@@ -2119,8 +2130,17 @@ fuse_session_new(struct fuse_args *args,
  * @return -errno  if failed to allocate memory to store `io`
  *
  **/
-int fuse_session_custom_io(struct fuse_session *se,
-				   const struct fuse_custom_io *io, int fd);
+#if (defined(LIBFUSE_BUILT_WITH_VERSIONED_SYMBOLS))
+/* Do not call this directly, but only through fuse_session_custom_io() */
+int _fuse_session_custom_io(struct fuse_session *se,
+			const struct fuse_custom_io *io,
+			size_t op_size, int fd);
+#else
+#define _fuse_session_custom_io(se, io, op_size, fd)	\
+	_fuse_session_custom_io_317(se, io, op_size, fd)
+#endif
+#define fuse_session_custom_io(se, io, fd)	\
```

Comment:
What is the purpose of this macro?  I had introduced the inlined function `fuse_session_new` which calls into `_fuse_session_new` to encode in the libfuse version an application was compiled against. _Maybe_ I should also do that for fuse_session_custom_io(). But right now this macro here does not bring an advantage?

Reply:
> What is the purpose of this macro? I had introduced the inlined function `fuse_session_new` which calls into `_fuse_session_new` to encode in the libfuse version an application was compiled against. _Maybe_ I should also do that for fuse_session_custom_io(). But right now this macro here does not bring an advantage?

Do you mean change to below:
```
@@ -2119,8 +2130,13 @@ fuse_session_new(struct fuse_args *args,
  * @return -errno  if failed to allocate memory to store `io`
  *
  **/ 
+#if (defined(LIBFUSE_BUILT_WITH_VERSIONED_SYMBOLS))
 int fuse_session_custom_io(struct fuse_session *se,
                   const struct fuse_custom_io *io, int fd);
+#else
+#define fuse_session_custom_io(se, io, fd) \
+   _fuse_session_custom_io_317(se, io, sizeof(*io), fd)
+#endif
```

But original fuse_session_custom_io() API doesn't support to pass sizeof(fuse_custom_io ) info, so for LIBFUSE_BUILT_WITH_VERSIONED_SYMBOLS branch, it won't work.

Reply:
I only see
```
#if  (defined(LIBFUSE_BUILT_WITH_VERSIONED_SYMBOLS))
int _fuse_session_custom_io(struct fuse_session *se,...
...
#endif
#define fuse_session_custom_io(se, io, fd) _fuse_session_custom_io(se, io, sizeof(*io), fd)
```

I.e. redirection of `fuse_session_custom_io` to the underscored function

Reply:
Oh, I think now I got what you mean.
Indeed we can change original fuse_session_custom_io() API to add sizeof(fuse_custom_io ) info for LIBFUSE_BUILT_WITH_VERSIONED_SYMBOLS branch, right?

Reply:
Hmm, seems that I was wrong, API must not be changed, Could you please give some code hints.